cf/data/array/cfanetcdfarray.py

```diff
@@ -17,75 +23,40 @@ class CFANetCDFArray(NetCDFArray):
 
     """
 
-    def __new__(cls, *args, **kwargs):
-        """Store fragment array classes.
-
-        .. versionadded:: 3.14.0
-
-        """
-        instance = super().__new__(cls)
-        instance._FragmentArray = {
-            "nc": NetCDFFragmentArray,
-            "um": UMFragmentArray,
-            None: MissingFragmentArray,
-        }
-        return instance
-
     def __init__(
         self,
         filename=None,
-        ncvar=None,
-        varid=None,
-        group=None,
+        address=None,
         dtype=None,
         mask=True,
         units=False,
         calendar=False,
         instructions=None,
+        substitutions=None,
+        term=None,
         source=None,
         copy=True,
+        x=None,
     ):
         """**Initialisation**
 
         :Parameters:
 
-            filename: `str`
-                The name of the netCDF file containing the array.
-
-            ncvar: `str`, optional
-                The name of the netCDF variable containing the
-                array. Required unless *varid* is set.
+            filename: (sequence of `str`), optional
+                The name of the CFA-netCDF file containing the
+                array. If a sequence then it must contain one element.
 
-            varid: `int`, optional
-                The UNIDATA netCDF interface ID of the variable
-                containing the array. Required if *ncvar* is not set,
-                ignored if *ncvar* is set.
+            address: (sequence of `str`0, optional
```

Comment:
```suggestion
            address: (sequence of `str`), optional
```

Reply:
Good catch - I realise that the `) should come after "of": 3fb241584ab0272c36082feec448ed3790f9d3a0